src/Controller/WopiController.php

```diff
@@ -88,9 +89,14 @@ protected function wopiCheckFileInfo(FileInterface $file, UserInterface $user, b
       'SupportsRename' => FALSE,
     ];
 
-    $user_picture = $user->user_picture?->entity;
-    if ($user_picture) {
-      $response_data['UserExtraInfo']['avatar'] = $this->fileUrlGenerator->generateAbsoluteString($user_picture->getFileUri());
+    if ($user->hasField('user_picture')) {
+      $user_picture = $user->get('user_picture')->entity;
+      if (
+        $user_picture instanceof FileInterface &&
```

Comment:
Instead of this instanceof we could just check `!== NULL` and then put a `/** @var */` doc or `assert()`.
Technically this instanceof also protects us from a redefined field that has a different type.

Reply:
Without instanceof or `@var`, phpstan will complain about `->getFileUri()` because that only exists for file entity.